Library/Equipment/GURPS Lite Equipment.eqp

```diff
@@ -0,0 +1,3453 @@
+<?xml version="1.0" encoding="US-ASCII" ?>
```

Comment:
As with the advantages, the word "Equipment" should be removed from the file's name